view/templates/dfrn_req_confirm.tpl

```diff
@@ -10,10 +10,10 @@
 <input type="hidden" name="dfrn_url" value="{{$dfrn_url}}" />
 <input type="hidden" name="confirm_key" value="{{$confirm_key}}" />
 <input type="hidden" name="localconfirm" value="1" />
-{{$aes_allow}}
+{{$aes_allow nofilter}}
 
 <label id="dfrn-request-homecoming-hide-label" for="dfrn-request-homecoming-hide">{{$hidethem}}</label>
-<input type="checkbox" name="hidden-contact" value="1" {{if $hidechecked}}checked="checked" {{/if}} />
```

Comment:
Why had you removed this here?

Reply:
The `$hidechecked` template variable was set to `''` in the only place where it is set, it is then useless.